resources/docker/images/delivery/tomcat/Dockerfile

```diff
@@ -53,7 +53,8 @@ RUN set -eux; \
     apk add --no-cache openssh-client; \
     apk add --no-cache git; \
 #    apk add --no-cache vim; \
-    apk add --no-cache nano;
+    apk add --no-cache nano; \
+    apk add --no-cache tini;
```

Comment:
I don't see `tini` used for this one, did I miss it?

Reply:
Updated